system/arch-linux-dell-precision-5570.md

```diff
@@ -0,0 +1,509 @@
+
+    Generate a configuration 
+
+        grub-mkconfig -o /boot/grub/grub.cfg
+
+    Find your devices UUID's
+
+        blkid
+
+    Verify by looking for UUID of your cryptroot and root inside. These are simple commands that will be run on boot time. Try to understand the logic in order to find where to look.
+        
+        vim -R grub.cnf
+
+12. Configure encrypted home with crypttab
+
+    > The /etc/crypttab (or, encrypted device table) file contains a list of encrypted devices that are to be unlocked when the system boots, similar to fstab. This file can be used for automatically mounting encrypted swap devices or secondary filesystems. It is read before fstab, so that dm-crypt containers can be unlocked before the filesystem inside is mounted. Note that crypttab is read after the system has booted, so it is not a replacement for unlocking via mkinitcpio hooks and boot loader options in the case of an encrypted root scenario. - [ArchWiki](https://wiki.archlinux.org/index.php/Dm-crypt/System_configuration#crypttab).
+
+    Find the encrypted home partition UUID.
+
+        > blkid
+        ...
+        /dev/sda4: UUID="00000a00-a0a0-0000-aa00-0aaa0a00aaaa" TYPE="crypto_LUKS" PARTLABEL="life" PARTUUID="..."
+        /dev/mapper/crypthome: UUID="..." TYPE="ext4"
+        ...
+
+    Copy the /dev/sdx4 UUID **not the /dev/mapper one** and paste it to the /etc/crypttab file.
+
+        > cat /etc/crypttab
+        # <name>       <device>                                     <password>              <options>
+        crypthome      UUID=00000a00-a0a0-0000-aa00-0aaa0a00aaaa    none                    luks,timeout=120
+
+    Finally, update the fstab to mount it automatically during the boot as follows:
+
+        > cat /etc/fstab
+        # <file system>			<mount point>   <type>  	<options>       	<dump>  <pass>
+        ...
+        /dev/mapper/crypthome		/home		ext4		defaults		    0       2
+        ...
+
+15. Last notes
+
+    If you're planning to use **wifi-menu** instead of the ethernet cable for further installations after booting into your system, you should install its dependencies right away.
+
+        pacman -S dialog wpa_supplicant iwd
+    
+    Change Root password since there is no default set:
+
+        chpwd root
+
+    Now you can safely exit the chroot, unmount your partitions and reboot into your new fresh system.
+
+        exit
+        umount /mnt/boot/efi
+        umount /mnt/boot
+        umount /mnt
+        reboot
+
+    Cross your fingers and hold tight, we hope you will enjoy your flight!
+
+
+<a name="configuring"></a>
+## Configuring with finess using Gnome 3
+
+Welcome to your new system!
+
+In this section, I will help you get started with a basic **G**raphical **U**ser **I**nterface and a basic system configuration using Gnome 3.
+
+1. Get your Dell Precision 5570 Arch drivers
+
+    Graphic card
+    
+        lspci | grep VGA
+        pacman -S xf86-video-intel
```

Comment:
@Tower450 update to dell ones from root logs

Reply:
```
pacman -S linux linux-api-headers linux-firmware linuxfirmware-whence linux-headers mesa mesa-utils nvidia-utils vulcan-intel
```

Reply:
- https://wiki.archlinux.org/title/Dell_Precision_5570
- https://wiki.archlinux.org/title/NVIDIA